plugins/default/lib/openshift.py

```diff
@@ -72,7 +72,8 @@ def _get_running_pod_containers(oc, project, selector=None):
     for p in pods:
         if p["status"]["phase"] == "Running":
             for c in p["spec"]["containers"]:
-                result.append((p["metadata"]["name"], c["name"], c))
+                if (not container_names or c["name"] in container_names):
+                    result.append((p["metadata"]["name"], c["name"]))
```

Comment:
Ah sorry, I didn't catch that in the rebase that you had added the extra parameter for the container object, will fix during rebase again and verify the cpu/memory checks are good.